README.md

```diff
@@ -82,6 +82,22 @@ Ubuntu 18.04     | 5.26
 Zonemaster requieres Perl version 5.14.2 or higher. Zonemaster has been 
 tested with the default version of Perl in the OSs as listed in the table above.
 
+### Supported Client Browser versions
+
+The Zonemaster GUI is tested against these configurations and should work perfectly with close configuration.
+
```

Comment:
"tested against these configurations" --> "tested against the browers below and for each browser the stated version and under the listed OS"

Is "close configuration" correct English? What does it mean?